src/neo4j_genai/types.py

```diff
@@ -48,7 +48,7 @@ class IndexModel(BaseModel):
     driver: Any
 
     @field_validator("driver")
-    def check_driver_is_valid(cls, v):
+    def check_driver_is_valid(cls, v: Any) -> neo4j.Driver:
```

Comment:
Is it possible to make this type hint v: neo4j.Driver?

Reply:
5ab1e8ca0179137388dc7b8386e5a7cbc0008d06